docs/options/rest_handlers.md

```diff
@@ -0,0 +1,132 @@
+# Rest Handlers
+
+This option is used to define the rest handlers that are not generated by UCC, i.e.
+custom handlers developed by the user.
+
+## Example
+
+```json
+{
+  "options": {
+    "restHandlers": [
+      {
+        "name": "StorageBuckets",
+        "endpoint": "Splunk_TA_Example_buckets",
+        "handlerType": "EAI",
+        "registerHandler": {
+          "file": "storage_buckets.py",
+          "actions": ["list", "create", "delete", "edit"]
+        },
+        "requestParameters": {
+          "create": {
+            "param_name": {
+              "required": true,
+              "schema": {
+                "type": "string"
+              }
+            }
+          },
+          "edit": {...},
+          "delete": {...},
+          "list": {...}
+        },
+        "responseParameters": {...}
+      }
+    ]
+  }
+}
+```
+
+The example above defines an `EAI` rest handler named `StorageBuckets` with the endpoint `Splunk_TA_Example_buckets`.
+UCC will modify the `web.conf` and `restmap.conf` files to register the handler. The handler file location is
+`bin/storage_buckets.py` and it will allow the actions `list`, `create`, `delete`, and `edit`.
+OpenApi entries will be generated for the handler with the specified request and response parameters. For example,
+`POST` requests to `.../Splunk_TA_Example_buckets` require a `param_name` parameter.
+
+## Required properties
+
+| Property    | Description                                                                                                                                     |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| name        | Name of the rest handler. Used in descriptions in the OpenApi specification. Only letters, digits and underscore.                               |
+| endpoint    | Endpoint of the rest handler: `https://localhost:PORT/servicesNS/-/<app_name>/<meta.restRoot>/<ENDPOINT>`. Only letters, digits and underscore. |
```

Comment:
We can change URL to `https://<SPLUNK_URL>:<MGMT_PORT>/servicesNS/-/<app_name>/<meta.restRoot>/<ENDPOINT>`, to keep it generic.